src/core/api/keyboard/keyboard_util.py

```diff
@@ -14,7 +13,6 @@
 from src.core.util.arg_parser import logger
 from src.core.util.system import shutdown_process
 
-logger = logging.getLogger(__name__)
```

Comment:
Don't delete the logger instance. Even if it's not being utilized in this code, it should be.